tensorboard/BUILD

```diff
@@ -288,6 +289,14 @@ py_library(
     visibility = ["//visibility:public"],
 )
 
+py_library(
+    name = "expect_absl_testing_absltest_installed",
+    # This is a dummy rule used as a absl-py dependency in open-source.
```

Comment:
As an aside, I wonder if perhaps we should just take an actual WORKSPACE dep on absl at this point.  We use a fair amount of it and unlike projects like TF and numpy that are either heavyweight or not natively built with bazel or both, absl-py is pretty light and already built with bazel.

Reply:
That seems reasonable. I’ll look into it.